### DISCUSSION TITLE
Can this be used to share devices remotely over a network?  What's the relationship to usbip-win(2)?

### DISCUSSION THREAD

**Discussion by sofakng** [dorssel/usbipd-win#1064] [General]
According to the description of this project, usbipd-win is used for "sharing locally connected USB devices to other machines" but then mentions Hyper-V and WSL 2.

Can this also be used to share devices remotely over a network between Windows or Windows <-> Linux machines?

There is also another project with a similar name (usbip-win and usbip-win2).  Does anybody know the differences?

**Comment by dorssel** [comment]
Yes, any (remote or virtual) machine with TCP network connectivity to port 3240 should work fine.

**Comment by Masterofowls** [comment]
=)
